core/alerts/alerts-core/src/main/java/com/thinkbiganalytics/alerts/api/core/AggregatingAlertProvider.java

```diff
@@ -298,7 +309,7 @@ public void alertsAvailable(int count) {
             final AtomicReference<DateTime> sinceTime = new AtomicReference<>(AggregatingAlertProvider.this.lastAlertsTime);
             Map<String, AlertSource> sources = snapshotAllSources();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "sources" which hides the field declared at line 86. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)